dev/sbt-plugin/src/sbt-test/sbt-plugin/run-all-javadsl/p/conf/application.conf

```diff
@@ -1,5 +1,4 @@
 lagom.play {
-  service-name="p"
   acls = [
     {
       path-regex="/p"
```

Comment:
Oh oh! This made me realize I didn't run scripted tests and there's code to be reviewed there.